pages/web_cloud/domains/dns_server_edit/guide.es-es.md

```diff
@@ -1,103 +1,214 @@
-updated: 2024-06-17
+title: "Modificar los servidores DNS de un dominio de OVHcloud"
+excerpt: "Descubra cómo modificar los servidores DNS de un dominio registrado en OVHcloud"
+updated: 2024-09-16
 ---
 
 > [!primary]
-> Esta traducción ha sido generada de forma automática por nuestro partner SYSTRAN. En algunos casos puede contener términos imprecisos, como en las etiquetas de los botones o los detalles técnicos. En caso de duda, le recomendamos que consulte la versión inglesa o francesa de la guía. Si quiere ayudarnos a mejorar esta traducción, por favor, utilice el botón «Contribuir» de esta página.
-> 
+> Esta traducción ha sido generada de forma automática por nuestro partner SYSTRAN. En algunos casos puede contener términos imprecisos, como en las etiquetas de los botones o los detalles técnicos. En caso de duda, le recomendamos que consulte la versión inglesa o francesa de la guía. Si quiere ayudarnos a mejorar esta traducción, por favor, utilice el botón "Contribuir" de esta página.
+>
 
 ## Objetivo
 
-La sigla DNS, que significa **D**omain **N**ame **S**ystem, es un conjunto de elementos (servidores DNS, zonas DNS, etc.) que permiten hacer coincidir un dominio con una dirección IP.
+Las siglas **DNS** (**D**omain **N**ame **S**ystem), son un conjunto de elementos (servidores DNS, zonas DNS, etc.) que permiten asociar un dominio a una dirección IP.
 
 Para más información, consulte nuestras guías "[Todo sobre los servidores DNS](/pages/web_cloud/domains/dns_server_general_information)" y "[Todo sobre la zona DNS](/pages/web_cloud/domains/dns_zone_general_information)".
 
-**Descubra cómo cambiar los servidores DNS de un dominio en OVHcloud.**
+**Esta guía explica cómo cambiar los servidores DNS de un dominio en 3 pasos.**
 
 <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/BvrUi26ShzI" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 ## Requisitos
 
 - Tener un [dominio](/links/web/domains) registrado en OVHcloud.
-- Disponer de [permisos adecuados para gestionar](/pages/account_and_service_management/account_information/managing_contacts) el nombre de dominio desde el [área de cliente de OVHcloud](/links/manager){.external}.
-- Estar conectado a su [área de cliente de OVHcloud](/links/manager){.external}.
+- Disponer de los permisos necesarios [para gestionar](/pages/account_and_service_management/account_information/managing_contacts) para el dominio desde el [área de cliente de OVHcloud](/links/manager).
+- Estar conectado a su [área de cliente de OVHcloud](/links/manager).
 
 > [!primary]
 >
-> Un **registrar** es una organización autorizada a vender dominios. OVHcloud forma parte de estos **registrars**.
+> Un **agente registrador** es una organización autorizada a vender dominios. OVHcloud forma parte de estos **agentes registradores**.
```

Comment:
not sure we should translate "registrar"

Reply:
corrected